starwars/src/App.js

```diff
@@ -30,11 +29,14 @@ class App extends Component {
     };
 
     render() {
-        return ( <
-            div className = "App" >
+        return ( < div className = "App" >
             <
-            h1 className = "Header" > React Wars < /h1> < /
-            div >
+            h1 className = "Header" > React Wars < /h1>     <
```

Comment:
You want the ```<``` character right before the ```h1``` instead of having it on another line. Also, you don't want space in the closing h1 tag - ```</h1>``` instead of ```< /h1>```.